lib/std/hash.zig

```diff
@@ -37,20 +37,85 @@ pub const XxHash3 = xxhash.XxHash3;
 pub const XxHash64 = xxhash.XxHash64;
 pub const XxHash32 = xxhash.XxHash32;
 
-/// This is handy if you have a u32 and want a u32 and don't want to take a
-/// detour through many layers of abstraction elsewhere in the std.hash
-/// namespace.
-/// Copied from https://nullprogram.com/blog/2018/07/31/
+/// Easy & fast hash function for integer types
```

Comment:
Please avoid words like "easy" and "fast" in doc comments. These are subjective feelings that do not belong in technical writing.

```suggestion
/// Integer to integer hashing for bit widths <= 256.
```